backend/src/main/java/gov/cdc/usds/simplereport/api/PersonNameResolver.java

```diff
@@ -2,11 +2,8 @@
 
 import gov.cdc.usds.simplereport.db.model.PersonEntity;
 import gov.cdc.usds.simplereport.db.model.auxiliary.PersonName;
-import graphql.kickstart.tools.GraphQLResolver;
 
 /** Resolver mix-in to handle aliasing "nameInfo" to "name" */
-public interface PersonNameResolver<T extends PersonEntity> extends GraphQLResolver<T> {
-  default PersonName getName(T entity) {
-    return entity.getNameInfo();
-  }
+public interface PersonNameResolver<T extends PersonEntity> {
+  PersonName getName(T entity);
```

Comment:
see [this comment](https://github.com/CDCgov/prime-simplereport/pull/4243/files#r966571178) for more details on this change